bot/src/commands/base.js

```diff
@@ -0,0 +1,17 @@
+export class BaseCommand {
+  constructor(api_url) {
+    this.api_url = api_url;
+  }
+
+  execute(author, channel, str, test=false) {
```

Comment:
Should this return a dummy empty `Promise.resolve()`? Just in case -- the calling code expects a Promise.